tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -974,6 +973,61 @@ public void SQLite_InstallAndLogMessage()
             }
         }
 
+        private void SetupSqliteConfigWithInvalidInstallCommand(string databaseName)
+        {
+            var nlogXmlConfig = @"
+            <nlog xmlns='http://www.nlog-project.org/schemas/NLog.xsd'
+                  xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' throwExceptions='false'>
+                <targets>
+                    <target name='database' xsi:type='Database' dbProvider='{0}' connectionstring='{1}' 
+                        commandText='insert into RethrowingInstallExceptionsTable (Message) values (@message);'>
+                        <parameter name='@message' layout='${{message}}' />
+                        <install-command ignoreFailures='false' text='THIS IS NOT VALID SQL;' />
```

Comment:
I explicitly set `ignoreFailures` to `false`, but I wonder if leaving it out (i.e. using the default) would be better? That way the test would fail if the default ever changes and give us a hint that we may need to update the `ThrowExceptions` doc?